sqlalchemy-stubs/sql/selectable.pyi

```diff
@@ -150,6 +150,10 @@ class FromGrouping(FromClause):
     def get_children(self, **kwargs: Any) -> Tuple[FromClause]: ...
     def __getattr__(self, attr: str) -> Any: ...
 
+_ValuesType = Optional[Union[Mapping[str, Any],
+                             Mapping[ColumnClause[Any], Any],
```

Comment:
I think you might need to also add `Mapping[Column[Any], Any]` to the union, otherwise the errors I found in our codebases will remain (since `Dict[Column[Any], Any]` is not a subtype of this because `Mapping` is invariant in key type).

Reply:
Oh, you said invariant and I interpreted it as covariant. Now I understand the issue. Maybe we should use `Mapping[Any, Any]` for now and then possibly create our own mapping type that is covariant.

Reply:
OK, It looks like `Mapping[Any, Any]` is the best option for now. Could you please also add a comment explaining why we don't use "more precise" types?